applications/fishing-map/src/features/workspace/workspace.selectors.ts

```diff
@@ -28,8 +28,24 @@ export const getDatasetsByDataview = (dataview: UrlDataviewInstance) =>
 
 export const selectWorkspace = (state: RootState) => state.workspace.data
 export const selectWorkspaceStatus = (state: RootState) => state.workspace.status
+export const selectWorkspaceError = (state: RootState) => state.workspace.error
 export const selectWorkspaceCustom = (state: RootState) => state.workspace.custom
 
+export const isWorkspacePublic = createSelector([selectWorkspace], (workspace) => {
+  return workspace?.public === true || workspace?.id.includes('-public')
```

Comment:
Again, sorry if I'm missing context, but haven't we said `public` would be a prefix? This might lead to errors if for some reason a workspace has `public` in its name but not the the actual public prefix.
```suggestion
  return workspace?.public === true || workspace?.id.substr(0,6) === 'public'
```
(if it's a suffix, same idea)

Reply:
Finally it is going to be a suffix, but thanks for catching false positives! I'll fix it